src/containers/Simularium/index.tsx

```diff
@@ -240,8 +240,13 @@ class App extends React.Component<AppProps, AppState> {
             simulariumController,
             changeToLocalSimulariumFile,
             conversionStatus,
+            embedFullscreen,
         } = this.props;
         const isEmbedded = location.pathname === EMBED_PATHNAME;
+        const hideSidePanels =
+            isEmbedded &&
+            window.innerWidth <= EMBEDDED_SIDE_PANEL_MINIMUM_WIDTH &&
+            !embedFullscreen;
```

Comment:
I'm not sure I understand all these conditionals. why is embedFullscreen an exception?